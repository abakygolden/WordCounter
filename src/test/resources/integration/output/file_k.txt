K 1
KAN 4
KEEP 2
KNOW 2
KONG 3
KONGEBES 1
KONGESANG 1
KR 1
KRITIK 1
KVINDER 1