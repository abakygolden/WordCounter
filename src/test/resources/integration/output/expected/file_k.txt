KEI 4
KEY 4
KIWIFRUIT 4
KUMQUAT 4